libraries.json

```diff
@@ -463,5 +463,10 @@
     "github": "Tetraquark/MyersDiffKt",
     "category": "Utils",
     "maven": "https://dl.bintray.com/tetraquark/kmplibs/ru/tetraquark/kmplibs/MyersDiffKt/"
+  },
+  {
+    "github": "microutils/kotlin-logging",
+    "category": "Logging",
+    "maven": "https://dl.bintray.com/microutils/kotlin-logging/io/github/microutils/"
```

Comment:
on this path not stored `maven-metadata.xml`. but it stored on `https://dl.bintray.com/microutils/kotlin-logging/io/github/microutils/kotlin-logging/` - please set this url

Reply:
Done.